.github/workflows/add-labels.yml

```diff
@@ -1,8 +1,8 @@
 name: Add Labels
 
 on:
-  pull_request:
-    branches: [ main ]
+  pull_request_target:
```

Comment:
So we used to do this. However, `pull_request_target` is a security risk. It exposes all secrets present in workflow, among other [bad things](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/). 🙈 

Thus we had to go back to using `pull_request` in https://github.com/ExpediaGroup/github-helpers/pull/232